_includes/manuals/nightly/1.2_release_notes.md

```diff
@@ -15,6 +15,7 @@ Foreman API resource listings support pagination. Foreman 2.2 introduces 'all' a
 ### Upgrade warnings
 
 * Trends and statistics are now moved to separate plugin. See [manual](/plugins/foreman_statistics) of the new Statistics plugin if you wish continue using it.
+* TFTP integration is now [disabled by default](https://community.theforeman.org/t/disable-foreman-proxy-tftp-support-by-default/18892). Upgrades are unaffected. To enable TFTP integration, `--foreman-proxy-tftp true` must be passed to the installer command.
```

Comment:
Just for the record, this hasn't been merged yet. https://github.com/theforeman/puppet-foreman_proxy/pull/602